common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1781,6 +1787,12 @@
       "Not allowed to implement multiple UDF interfaces, UDF class <className>."
     ]
   },
+  "NAMED_ARGUMENTS_NOT_SUPPORTED" : {
+    "message" : [
+      "Named arguments are not supported for function <functionName>; please retry the query with positional arguments to the function call instead or enable the feature using the SQL config \"spark.sql.allowNamedFunctionArguments\"."
```

Comment:
I don't think this error is triggered when the config is false right? It is only because the function itself doesn't implement the named argument trait.

Reply:
Actually, you are right. There is a separate exception for this.